api/src/server.ts

```diff
@@ -28,6 +49,68 @@ export class Server {
         if (registrationError) {
           reject(registrationError);
         } else {
+          this.server.auth.strategy("jwt", "jwt", "required", {
+            complete: true,
+            key: require("jwks-rsa").hapiJwt2Key({
+              cache: Auth0.jks.cache,
+              rateLimit: Auth0.jks.rateLimit,
+              jwksRequestsPerMinute: Auth0.jks.requestsPerMinute,
+              jwksUri: Auth0.jks.uri,
+            }),
+            // TODO Ideally pull from DI
+            //key: Container.get<IKeyLocator>(Identifiers.KeyLocator),
```

Comment:
this is where we can swap out how we validate the jwt token.  The `hapiJwt2Key` works fine for auth0 but for tests we may need a slightly different strategy since a `jwksUri` won't be available.  In tyco subscription service I leveraged DI to swap this in/out